test/common/markdown/markdown.tests.ps1

```diff
@@ -74,6 +74,7 @@ Describe 'Common Tests - Validate Markdown Files' -Tag 'CI' {
             $docsToTest = @(
                 './*.md'
                 './docs/*.md'
+                './docs/cmdlet-example/command-line-simple-example.md'
```

Comment:
Maybe we can change this to `'./docs/cmdlet-example/*.md'`, so that #5096 doesn't need to edit `markdown.tests.ps1` again.